docs/copied-from-beats/shared-template-load.asciidoc

```diff
@@ -7,14 +7,14 @@
 //// resolve Beat names: beatname_uc and beatname_lc
```

Comment:
[Link](https://www.elastic.co/guide/en/apm/server/6.5/_manually_loading_template_configuration.html). There are a lot of changes in this file that I imagine I can test myself - just need someone to show me what that might look like.

Reply:
this all looks fine :+1: